### PR TITLE
Use formatting to get github actions annotations

### PR DESCRIPTION
**Pull request by foursixnine**

### CODE REVIEW

File: .github/workflows/ci-tests.yaml [hunk 1]
```diff
@@ -33,7 +33,8 @@ jobs:
         run: perl -V
       - name: Set up perl
         run: |
-          cpanm --with-develop --installdeps .
+          cpanm IO::Scalar # skip missing dependency on TAP::Harness::GithubActions
```
Comment (foursixnine) [inline .github/workflows/ci-tests.yaml:36]: Gotta wait for the next release of TAP::Harness::GithubActions

**Review by okurz — COMMENTED** [review]
+1

**Comment by foursixnine** [comment]
Yisus christ, @codecov likes to get on my nerves:

```
400 Bad Request
Could not determine repo and owner
```

**Comment by foursixnine** [comment]
I'm calling this done, next step, generating always the Build.PL instead of using a customized one (and adding a `feature release` to the cpanfile

**Review by Martchus — APPROVED** [review]

File: Build.PL [hunk 2]
```diff
@@ -14,6 +14,12 @@ use builder::custom;
 use File::Basename;
 use File::Spec;
 
+my $tap_harness_args = {
+    "verbosity" => 1,
+    "merge" => 1,
+    "formatter" => "TAP::Formatter::GitHubActions",
```
Comment (josegomezr) [inline Build.PL:20]:
```suggestion
    "formatter_class" => "TAP::Formatter::GitHubActions",
```

`formatter` key expects a `TAP::Formatter::*` object as value.

File: Build.PL [hunk 3]
```diff
@@ -14,6 +14,13 @@ use builder::custom;
 use File::Basename;
 use File::Spec;
 
+my $tap_harness_args = {
```
Comment (perlpunk) [inline Build.PL:17]: Why is it necessary to do this in the Build.PL? Then everyone installing this from CPAN would have to install TAP::Formatter::GitHubActions. But we only need this output in the CI, right?

Comment (foursixnine) [reply Build.PL:17]: > Why is it necessary to do this in the Build.PL? Then everyone installing this from CPAN would have to install TAP::Formatter::GitHubActions. But we only need this output in the CI, right?

It's one of the reasons why I haven't generated a release yet, I want to make the tests pass first, and addressing the generation of build.pl by minilla: (so only on CI we'll need the actions) 

> But we only need this output in the CI, right?

Right :)

**Pull request merged**